packages/dev/core/src/Maths/math.plane.ts

```diff
@@ -189,10 +201,9 @@ export class Plane {
      * Note : the vector "normal" is updated because normalized.
      */
     static FromPositionAndNormal(origin: DeepImmutable<Vector3>, normal: Vector3): Plane {
-        const result = new Plane(0.0, 0.0, 0.0, 0.0);
         normal.normalize();

```

Comment:
actually, as @Popov72 told me offline it would be great instaed to keep FromPositionAndNorma and add a static FromPositionAndNormalToRef instead of upgrading the instance function.

It will help tree shaking better if we can one day move to a side effect free package.